app/views/exam_templates/_student_info.html.erb

```diff
@@ -24,9 +24,17 @@
 
   <div id="exam-cover-display-<%= exam_template.id %>" class="<%= exam_template.automatic_parsing ? 'flex-display' : 'no-display' %>">
     <div id="exam_cover-<%= exam_template.id %>">
-      <img src="<%= show_cover_course_exam_template_path(@current_course, exam_template) %>" class="crop-target"
-           alt="<%= t(:'exam_templates.parsing.cover_page_for', id: exam_template.id) %>"
-           loading="lazy">
+      <div class="exam-crop-container">
+        <div class="exam-crop-img-container">
+          <img src="<%= show_cover_course_exam_template_path(@current_course, exam_template) %>" class="crop-target"
+               alt="<%= t(:'exam_templates.parsing.cover_page_for', id: exam_template.id) %>"
+               loading="lazy">
+        </div>
+        <div class="exam-crop-button-container">
+          <button id="increase-crop-scale" class="crop-scale-button" type="button">+</button>
```

Comment:
add a `title` attribute and set it to be the same (localized) "Zoom in" and "zoom out" strings that the image viewer uses